Doc/library/ssl.rst

```diff
@@ -1701,13 +1701,15 @@ to speed up repeated connections from the same clients.
    :meth:`~SSLContext.wrap_socket` in order to match the hostname.  Enabling
    hostname checking automatically sets :attr:`~SSLContext.verify_mode` from
    :data:`CERT_NONE` to :data:`CERT_REQUIRED`.  It cannot be set back to
-   :data:`CERT_NONE` as long as hostname checking is enabled.
+   :data:`CERT_NONE` as long as hostname checking is enabled. The
+   :data:`PROTOCOL_TLS_CLIENT` protocol enables hostname checking by default.
+   With other protocols, hostname checking must be enabled explicitly.
 
    Example::
 
       import socket, ssl
 
-      context = ssl.SSLContext()
+      context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
```

Comment:
Doesn't this preclude TLS 1.3?

Reply:
Yes, but it's just an example. I'd like to include an example that shows how to make protocols other than ``PROTOCOL_TLS_CLIENT`` secure. TLS 1.2 is good enough.

Note, there won't be a TLS 1.3 protocol.

Reply:
Okay, that makes sense.